backend/src/database/repositories/memberSegmentAffiliationRepository.ts

```diff
@@ -152,6 +154,28 @@ class MemberSegmentAffiliationRepository extends RepositoryBase<
 
     return records
   }
+
+  private async updateAffiliation(memberId, segmentId, organizationId) {
+    const transaction = this.transaction
+
+    const query = `
+      UPDATE activities
+      SET "organizationId" = :organizationId
+      WHERE "memberId" = :memberId
+        AND "segmentId" = :segmentId
+        AND "organizationId" IS NULL
```

Comment:
why only null ones - So when a member's affiliation was previously set, does this mean that we can't change it again back to some other org?